src/Feather.jl

```diff
@@ -1,5 +1,11 @@
 module Feather
 
+if v"0.6" <= VERSION < v"0.7-DEV"
```

Comment:
No need for the lower bound, the package requires 0.6 anyway. Also better use the exact commit in which the changed happened, which is `0.7.0-DEV.2009"`.